### PR TITLE
:robot: Add outflank-c2-collection.vm

### PR DESCRIPTION
**Pull request by vm-packages**
Automated PR from https://github.com/mandiant/VM-Packages/issues/382 :rocket:

### CODE REVIEW

**Comment by day1player** [comment]
@nos3curity this category should be Information Gathering

**Comment by day1player** [comment]
Also recommend updating package name to "outflank-c2-tool-collection" (I think "tool" is missing currently).

**Comment by nos3curity** [comment]
@day1player changed package name and category. Ready to merge if you approve.

File: packages/outflank-c2-tool-collection.vm/outflank-c2-tool-collection.vm.nuspec [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>outflank-c2-collection.vm</id>
```
Comment (day1player) [inline packages/outflank-c2-tool-collection.vm/outflank-c2-tool-collection.vm.nuspec:4]: please add "tool"

Comment (nos3curity) [reply packages/outflank-c2-tool-collection.vm/outflank-c2-tool-collection.vm.nuspec]: @day1player added. Merge when approved.

**Pull request merged**